README.md

```diff
@@ -40,55 +36,17 @@ http://localhost:3000
 
 Note - Frontend starts on port 3000 and simple Node.js server also starts on port 3001. If you need to change the ports, configure in `package.json`, `proxy-server.js`, `src/constants` 
 
-#### Setting up with Docker
-
-You can build a Docker image of the JanusGraph visualizer with the included `Dockerfile`.
-This will use the current version of the `main` branch of the source GitHub repository.
-The Docker image can be built by calling the `docker build -f full.Dockerfile` command, for example:
-
-```sh
-docker build --tag=janusgraph-visualizer:latest -f full.Dockerfile .
-```
-
-If you had already built node project on your host then you can create a Docker image faster by using `Dockerfile` instead of `full.Dockerfile`: 
-
-```sh
-docker build --tag=janusgraph-visualizer:latest .
-```
-
-The image can also be downloaded from Docker hub: [`janusgraph/janusgraph-visualizer:latest`](https://hub.docker.com/r/janusgraph/janusgraph-visualizer).
-
-```sh
-docker pull janusgraph/janusgraph-visualizer:latest
-```
-
-The Docker image can then be run by calling `docker run` and exposing the necessary ports for communication. See [Docker's documentation](https://docs.docker.com/engine/reference/commandline/run/) for more options on how to run the image.
-
-```sh
-# if you built the image yourself
-docker run --rm -d -p 3000:3000 -p 3001:3001 --name=janusgraph-visualizer --network=host janusgraph-visualizer:latest
-# if you downloaded from Docker Hub
-docker run --rm -d -p 3000:3000 -p 3001:3001 --name=janusgraph-visualizer --network=host janusgraph/janusgraph-visualizer:latest
-```
-Note that `--network=host` is not needed if you don't run your gremlin server in the host machine. 
-
-* Open the browser and navigate to
-```sh
-http://localhost:3001
-```
-
-The Docker container can be stopped by calling `docker stop janusgraph-visualizer`.
-
 ### Supported Environment Variables
 
-* `GREMLIN_HOST` - sets gremlin server hostname for connection. Default is `localhost`.
+* `GREMLIN_HOST` - sets gremlin server hostname for connection. Default is `janusgraph`.
```

Comment:
The default value is still `localhost`, but when started via `docker-compose` then the environment variable `GREMLIN_HOST` is supplied with `janusgraph`. If people start it directly with `docker run` then the default value will be `localhost` and not `janusgraph`. So this may be confusing. Maybe we should say that the default value is `localhost`, but the supplied `docker-compose.yml` will use `janusgraph` instead. Maybe it's not even needed to be specified because people who will use `docker-compose.yml` and will want to communicate with the `GREMLIN_HOST` will use it's hostname anyways.
So, my suggestion is to either leave it as before or add a clarification comment.